package.json

```diff
@@ -62,7 +62,7 @@
     "better-than-before": "1.0.0",
     "chai": "3.4.1",
     "chokidar-cli": "1.2.0",
-    "conventional-changelog-core": "1.0.1",
+    "conventional-changelog-core": "1.8.0",
```

Comment:
Relevant release that added the feature that's broken here.

https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-core/CHANGELOG.md#150-2016-05-10